### PR TITLE
Adding sample of splitting table, Merge documents with destination margins.

### PR DESCRIPTION
**Pull request by sharonaumao**
Added KB documentation sample. The samples are:

- Split-table-with-same-format
- Merge-documents-with-destination-margin

### CODE REVIEW

File: Word-document/Merge-documents-with-same-margins/.NET-Standard/Merge-documents-with-same-margins/Program.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                        //Open the destination document.
+                        using (WordDocument destinationDocument = new WordDocument(destinationStreamPath, FormatType.Automatic))
+                        {
+                            //Get the page setup of the destination document.
+                            WPageSetup destinationDocumentPageSetup = destinationDocument.LastSection.PageSetup;
+                            //Iterate source document.
+                            foreach (WSection sourceSection in sourceDocument.Sections)
+                            {
+                                sourceSection.PageSetup.Margins = destinationDocumentPageSetup.Margins;
+                                //Clone the source document sections to the destination document.
```
Comment (Ramaraj-Marimuthu) [inline Word-document/Merge-documents-with-same-margins/.NET-Standard/Merge-documents-with-same-margins/Program.cs:28]: @sharonaumao - Please modify the XML comments properly.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Word-document/Merge-documents-with-same-margins/.NET-Standard/Merge-documents-with-same-margins/Program.cs (hunk 1 above)
Comment (sharonaumao) [reply Word-document/Merge-documents-with-same-margins/.NET-Standard/Merge-documents-with-same-margins/Program.cs]: @Ramaraj-Marimuthu -I have modified

File: Word-document/Merge-documents-with-destination-margin/.NET-Standard/Merge-documents-with-destination-margin/Program.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                        //Open the destination document.
+                        using (WordDocument destinationDocument = new WordDocument(destinationStreamPath, FormatType.Automatic))
+                        {
+                            //Get the page setup of the destination document.
+                            WPageSetup destinationDocumentPageSetup = destinationDocument.LastSection.PageSetup;
+                            //Iterate source document.
+                            foreach (WSection sourceSection in sourceDocument.Sections)
+                            {
+                                sourceSection.PageSetup.Margins = destinationDocumentPageSetup.Margins;
+                                //Clone and copy the source document sections to the destination document.
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply Word-document/Merge-documents-with-destination-margin/.NET-Standard/Merge-documents-with-destination-margin/Program.cs]: @Ramaraj-Marimuthu -Modified it sir

**Review by Ramaraj-Marimuthu — APPROVED** [review]
I don't have any concerns in this sample changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**